css/welcome.css

```diff
@@ -94,35 +94,19 @@ main {
 #welcome-nav a {
   position: absolute;
   width: 13vmin;
-  height: 17vmin;
-  background: white;
-  border-radius: 5px;
-  border: 0.5vmin solid gray;
+  height: calc(13vmin*1.3975);
```

Comment:
(nit) add a comment explaining this is a hack to get the a tag's size to be the same as the card image